ballerina-shell/modules/shell-cli/src/main/resources/command.help.txt

```diff
@@ -0,0 +1,22 @@
+Ballerina Shell Help
+
+Type a ballerina statement, expression or a declaration.
+
+Imports should be done with prefixes and the modules
+should be available in the classpath.
+To see available imports use /imports.
+
+Following commands are the in-built commands.
+
+/exit           - Exit the terminal shell.
+/debug          - Toggle debug mode. You will see debug messages.
+/reset          - Reset REPL state.
+                  All previous code snippets will be removed from REPL memory.
+/imports        - List the available imports.
+/vars           - List the declared variables and their values.
+/dclns          - List the user defined module level declarations.
+/help           - Get this message.
+/help <TOPIC>   - Get help on <TOPIC>. You may type in a topic to search for
```

Comment:
Do we allow arbitrary text also here with <TOPIC>? Or is it only the available commands? Then we need to fix the wording here.

Reply:
There are a set of available commands but there are a lot of available prefixes. (basically, this command will search for the topic in BBE with the same name and give the output)

I added `help topics` to see available topics. So how about this?

```
Ballerina Shell Help

Type a Java language expression, statement, or declaration.
Or type one of the following commands:

/exit           - Exit the terminal shell.
/debug          - Toggle debug mode. You will see debug messages.
/reset          - Reset REPL state.
                  All previous code snippets will be removed from REPL memory.
/imports        - List the available imports.
/vars           - List the declared variables and their values.
/dclns          - List the user defined module level declarations.
/help           - Get this message.
/help <TOPIC>   - Get help on <TOPIC>. eg: /help var
/help topics    - See all available topic names.
```

Reply:
Is it actually correct to call it TOPIC? That is my question. Can you post some examples and their output here?

Reply:
```
=$ /help var                                                                                                                                    
| Ballerina allows you to declare variables using the `var` keyword instead of specifying a type.
| The type of the declared variable is inferred from the expression on the right-hand side.

=$ /help enums                                                                                                                                   
| An enum member is defined as a constant. An enum member can explicitly specify a constant expression
| as an associated value. If specified, the result of evaluating the relevant expression must be a string. If the
| constant expression is omitted, it defaults to be the same as the identifier.
```

The current help provider is a simple implementation that basically fetches the entry from BBE for the word given.